src/jukebox/jukebox/speaking_text/__init__.py

```diff
@@ -0,0 +1,67 @@
+"""
+Text to Speech. Plugin to speak any given text via speaker
+"""
+
+import logging
+import jukebox.cfghandler
+import shlex
+import subprocess
+
+logger = logging.getLogger('jb.speaking_text')
+cfg = jukebox.cfghandler.get_handler('jukebox')
+
+
+def run_unix_command(command):
+    cmds = shlex.split(command)
+    return subprocess.Popen(cmds, start_new_session=True)
+
+
+# http://espeak.sourceforge.net/commands.html
+def with_espeak(text, params):
+    # Language
+    lang = cfg.getn('speaking_text', 'lang') or 'en'
+    if 'lang' in params:
+        lang = params['lang']
+
+    # Speak punctuation?
+    speakPunct = ''
+    speakPunctOption = cfg.getn('speaking_text', 'speakPunct') or False
+    if 'speakPunct' in params:
+        speakPunctOption = params['speakPunct']
+
+    if speakPunctOption:
+        speakPunct = '--punct="<characters>"'
+
+    # speed in words-per-minute
+    speedOption = cfg.getn('speaking_text', 'speed') or 'normal'
+    if 'speed' in params:
+        speedOption = params['speed']
```

Comment:
I would allow a numerical value here, that someone could use the full feature set of espeak

Reply:
My last commits allow this now too